src/containers/TeamInfo.jsx

```diff
@@ -114,45 +118,47 @@ function TeamInfo(props) {
         deletionClone.splice(i, 1);
       }
     }
+
+    fetch(`/deleteTeam/${teamInfo.team_id}`, {
+      method: 'DELETE',
+    }).catch((err) => console.log(err));
+
     alert('Team has been deleted!');
     props.sync(deletionClone);
-    navigate('/home')
-  }
+    navigate('/home');
+  };
 
   // Populate team members + activities
-  const teamMembers = teamInfo.teamMembers.map(ele =>
+  const teamMembers = teamInfo.teamMembers.map((ele) => (
```

Comment:
You cleaned up extra parens above but wrote them in here?

Reply:
That's probably my prettier going off on auto-formatting when I save